modules/custom/az_accordion/src/Plugin/Field/FieldFormatter/AZAccordionDefaultFormatter.php

```diff
@@ -134,7 +134,7 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
         '#accordion_item_id' => Html::getUniqueId('az_accordion'),
         '#collapsed' => $item->collapsed ? 'collapse' : 'collapse show',
         '#aria_expanded' => !$item->collapsed ? 'true' : 'false',
-        '#aria_controls' => Html::getUniqueId('az_accordion_aria_controls'),
+        '#aria_controls' => Html::getUniqueId('az_accordion'),
```

Comment:
Maybe set the ID as a variable and then using that variable in the two places it is needed.
#aria_controls'
and
'#accordion_item_id'

Reply:
I was thinking the same thing.  I also wonder if we could potentially do away with (or possibly deprecate) the `aria_controls` variable in the template since it seems like we can just use the same ID value in the `accordion_item_id` template variable.

(See https://github.com/az-digital/az_quickstart/blob/issue/3792/modules/custom/az_accordion/templates/az-accordion.html.twig)

Reply:
(Deprecating that template variable could be a follow-up improvement separate from the bugfix)

Reply:
I sent the initial changes to Jeff and he seemed to not have any issue with it. Might also be good to talk about the mismatched IDs regardless.

Reply:
Made follow-up issue for deprecating the variable: #3807